doc/install/build-ceph.rst

```diff
@@ -67,9 +67,9 @@ Debian/Ubuntu or ``rpmbuild`` for the RPM Package Manager.
 Advanced Package Tool (APT)
 ---------------------------
 
-To create ``.deb`` packages for Debian/Ubuntu, ensure that you have cloned the 
-`Ceph`_ repository, installed the `Build Prerequisites`_ and installed 
-``debhelper``::
+To create ``.deb`` packages for Debian/Ubuntu, ensure that you have cloned the
+`Ceph`_ repository, installed the `Build Prerequisites`_, run do_cmake.sh, and
```

Comment:
`do_cmake.sh` tries to apply different parameters passed to cmake based on the operating system and GNU/Linux disto installed on the building host.

and it uses a different build directory than that of dpkg-buildpackage and rpm-build. so i don't think this change is correct.

Reply:
Thank you for the clarification!  I don't yet know what the right fix is, but the documentation is lacking in its current form -- following it does not result in packages being built.  I will close this PR and I will do another PR if I figure out what the correct fix is.